tests/units/states/test_state_base.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from abc import ABCMeta
+
+from urnai.states.state_base import StateBase
+
+
+class TestStateBase(unittest.TestCase):
+
+    def test_abstract_methods(self):
+        StateBase.__abstractmethods__ = set()
+
+        class FakeState(StateBase):
+            def __init__(self):
+                super().__init__()
+
+        f = FakeState()
+        update_return = f.update("observation")
+        state = f.get_state
+        dimension = f.get_dimension
```

Comment:
Ideally, it shouldn't be `get_<something>`, but just the name of the attribute. Example: instead of `f.get_state` it would be `f.state`.